INSTALL.md

```diff
@@ -96,50 +68,55 @@ apt-get install hip-nvcc
    * By default HIP is installed into /opt/rocm/hip (can be overridden by setting HIP_PATH environment variable).
    * Optionally, consider adding /opt/rocm/bin to your path to make it easier to use the tools.
 
+# Building HIP from source 
 
-## Verify your installation
-Run hipconfig (instructions below assume default installation path) :
-```shell
-/opt/rocm/bin/hipconfig --full
-```
-
-Compile and run the [square sample](https://github.com/ROCm-Developer-Tools/HIP/tree/master/samples/0_Intro/square). 
+## Build ROCclr
 
+ROCclr is defined on AMD platform that HIP use Radeon Open Compute Common Language Runtime (ROCclr), which is a virtual device interface that HIP runtimes interact with different backends.
```

Comment:
Radeon Open Compute Common Language Runtime (ROCclr) is a virtual device interface that HIP Runtimes interact with different backends.